.gitignore

```diff
@@ -22,8 +22,12 @@
 /vendor/
 /Godeps/
 
+### Go Workspace ###
+go.work
+go.work.sum
```

Comment:
this is optional workspace files to support multiple modules in one project, I used them to build latest protoc_gen_go and golangci-lint

if you want, I can add my build config here to to show what I mean